packages/vuetify/src/components/VInput/VInput.ts

```diff
@@ -66,7 +66,7 @@ export default baseMixins.extend<options>().extend({
     classes (): object {
       return {
         'v-input--has-state': this.hasState,
-        'v-input--hide-details': this.hideDetails,
+        'v-input--hide-details': this.genMessages() === null,
```

Comment:
Not too stoked about using a render method here.

Reply:
Fixed